### PR TITLE
Diagnostics with tracing

### PR DESCRIPTION
**Pull request by adzialocha**

### CODE REVIEW

File: org.p2panda.aardvark.json [hunk 1]
```diff
@@ -46,6 +46,9 @@
                     "url" : "./",
                     "branch" : "HEAD"
                 }
+            ],
+            "config-opts" : [
+                "--libdir=lib"
```
Comment (swick) [inline org.p2panda.aardvark.json:51]: what is this for?

Comment (adzialocha) [reply org.p2panda.aardvark.json:51]: I dont know! Builder added that automatically 😐

Comment (swick) [reply org.p2panda.aardvark.json:51]: oh, it does the same for me... let's just go with it.

**Comment by swick** [comment]
LGTM

**Comment by adzialocha** [comment]
This is not working yet as I cant get the env vars into the build process. Setting them in the builder config doesn't have any effect.

Im suspecting meson to not forward these settings but Im also a bit overasked ..

**Comment by swick** [comment]
So, adding env to the build context really only adds them during the build. We can set the env for execution in finish-args:
```patch
diff --git ./org.p2panda.aardvark.json ../org.p2panda.aardvark.json
index bcbbd31..8d03175 100644
--- ./org.p2panda.aardvark.json
+++ ../org.p2panda.aardvark.json
@@ -12,17 +12,15 @@
         "--share=ipc",
         "--socket=fallback-x11",
         "--device=dri",
-        "--socket=wayland"
+        "--socket=wayland",
+        "--env=RUST_BACKTRACE=1",
+        "--env=RUST_LOG=debug"
     ],
     "build-options" : {
         "append-path" : "/usr/lib/sdk/rust-stable/bin",
         "build-args" : [
             "--share=network"
-        ],
-        "env" : {
-            "RUST_BACKTRACE" : "1",
-            "RUST_LOG" : "debug"
-        }
+        ]
     },
     "cleanup" : [
         "/include",
~
```

That works, but also means the app would ship to users like that. Ideally one could configure env in builder (Configure Project -> Applications). But unfortunately I can only choose the Run Command (Flatpak Application) but not set any environment.

Another way is to use a Command (Configure Project -> Commands) to spawn e.g. bash with "As Target Application" Locality and then just run "RUST_DEBUG=1 aardvark" there...

**Comment by swick** [comment]
rebased

**Pull request merged**